.editorconfig

```diff
@@ -0,0 +1,20 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
```

Comment:
out of interest - is this really needed or isn't this the default anyway?

Reply:
Don't thinks it needed since we all use a sane platform, but I think if it's not there it just won't be given to the editor, meaning system defaults. Can remove it.

Reply:
nah, not worth the churn, just leave it. Thanks!